packages/devtools/lib/src/memory/memory.dart

```diff
@@ -77,12 +83,19 @@ class MemoryScreen extends Screen with SetStateMixin {
 
   PButton gcNowButton;
 
-  ListQueue<Table<Object>> tableStack = ListQueue<Table<Object>>();
+  ListQueue<Table<dynamic>> tableStack = ListQueue<Table<dynamic>>();
 
   MemoryChart memoryChart;
 
   CoreElement tableContainer;
 
+  bool isTreeTable(Table<dynamic> table) => (table is TreeTable) ? true : false;
+
+  TreeTable<dynamic> getTreeTable(Table table) =>
```

Comment:
Should this be top level? Or static?

Is this used anywhere?

Also on naming - https://dart.dev/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get